### DISCUSSION TITLE
What is the expected behavior for seealso tag in a class doc comment

### DISCUSSION THREAD

**Discussion by pepone** [dotnet/docfx#8724] [General]
When I add a seealso tag to the class doc comment I see this rendered with the sealso section of each method in the class, and also in the seealso section of the class, for example:

```csharp
/// <summary>A nice class</summary>
/// <seealso cref="MyOtherClass"/>
public class MyClass
{
    /// <summary>A nice operation similar to <see cref="MyOtherClass.MyOtherOperation(string, int)"/>.</summary>
    public void MyOperation() {}

    /// <summary>Another nice operation</summary>
    public void MoreOperations() {}
}
```

<img width="416" alt="seealso-in-class" src="https://user-images.githubusercontent.com/254604/236209551-38a9d907-4411-4e74-924c-4c0594b88b25.png">

Is this the expected behavior? seems odd to repeat the same link over and over.

**Comment by yufeih** [comment]
No it isn't. This probably yet another bug caused by mustache cascade variable lookup.
